src/main/java/cs361/battleships/models/Square.java

```diff
@@ -19,6 +19,19 @@ public Square(int row, char column) {
 		this.column = column;
 	}
 
+	public void move(char moveDir) {
+		switch(moveDir) {
```

Comment:
good use of a switch statement for each of the directions. Im not sure that there is really a better way to store the individual directions except for maybe an array